packages/device_info/lib/device_info.dart

```diff
@@ -0,0 +1,231 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
```

Comment:
Definitely! So far I've been focusing on making the building blocks available in order to unblock users who need _some kind of_ crash reporting asap. Near-zero effort has been spent on making it convenient though. We are collecting issues in [this project](https://github.com/flutter/flutter/projects/14?fullscreen=true) to make our crash reporting story more complete.